scripts/pcm-dpc/covid-19/dpc-covid19-ita_StatisticalVariable.mcf

```diff
@@ -0,0 +1,95 @@
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:PatientHospitalizedWithSymptoms
+
+Node: dcid:Count_MedicalConditionIncident_COVID_19_PatientInICU
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:PatientInICU
+
+Node: dcid:Count_MedicalConditionIncident_COVID_19_PatientHospitalized
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:PatientHospitalized
+
+Node: dcid:Count_MedicalConditionIncident_COVID_19_PatientInHomeIsolation
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:PatientInHomeIsolation
+
+Node: dcid:Count_MedicalConditionIncident_COVID_19_ActiveCase
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:ActiveCase
+
+Node: dcid:IncrementalCount_MedicalConditionIncident_COVID_19_ActiveCase
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+statType: dcs:measuredValue
+medicalStatus: dcs:ActiveCase
+
+Node: dcid:CumulativeCount_MedicalConditionIncident_COVID_19_PatientRecovered
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:cumulativeCount
+statType: dcs:measuredValue
+medicalStatus: dcs:PatientRecovered
+
+Node: dcid:CumulativeCount_MedicalConditionIncident_COVID_19_PatientDeceased
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:cumulativeCount
+statType: dcs:measuredValue
+medicalStatus: dcs:PatientDeceased
+
+Node: dcid:CumulativeCount_MedicalConditionIncident_COVID_19_PositiveCase
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:cumulativeCount
+statType: dcs:measuredValue
+medicalStatus: dcs:PositiveCase
+
+Node: dcid:IncrementalCount_MedicalConditionIncident_COVID_19_PositiveCase
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+statType: dcs:measuredValue
+medicalStatus: dcs:PositiveCase
+
+Node: dcid:CumulativeCount_MedicalTest_COVID_19
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicaltTest
+incidentType: dcs:COVID_19
+measuredProperty: dcs:cumulativeCount
+statType: dcs:measuredValue
+
+Node: dcid:CumulativeCount_Person_COVID_19_Tested
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Person
+incidentType: dcs:COVID_19
```

Comment:
This should change to medicalCondition.

Reply:
To confirm, do you mean change "incidentType" to "medicalCondition"?

Reply:
Done